webapp/src/ts/services/cht-datasource.service.ts

```diff
@@ -121,6 +121,9 @@ export class CHTDatasourceService {
         },
         getExtensionLib: (id) => {
           return this.extensionLibs[id];
+        },
+        context: {
+          targetDocs: []
         }
```

Comment:
This isn't very clear to me. When looking at dataSource, it's about exposing functions on scope, for sample `[person](https://github.com/medic/cht-core/blob/94ac934a05dfa1b1ebe35d4ffa6fe7673179ad9d/shared-libs/cht-datasource/src/index.ts#L74)`. 
It's unclear to me what entity this `context`  is about. Also, why is there a `targetDocs: []` directly instead of having an accessor function?

Reply:
This is the design choice we made for the feature release. There have been lots of requests over time to add more data to contact summary. 
It's not an accessor function because the new code replaces how we get the target that always gets passed and was part of the original interface, so the computation happens anyway. We can make it an accessor function, but it would just be candy.

Reply:
It doesn't seem fit like that 🤔. [Looking at the documentation:](https://github.com/medic/cht-core/blob/94ac934a05dfa1b1ebe35d4ffa6fe7673179ad9d/shared-libs/cht-datasource/src/index.ts#L4)

> This module provides a simple API for interacting with CHT data.

Looks like it needs a [target file here](https://github.com/medic/cht-core/tree/94ac934a05dfa1b1ebe35d4ffa6fe7673179ad9d/shared-libs/cht-datasource/src) to return the list of target [docs type](https://github.com/medic/cht-core/blob/94ac934a05dfa1b1ebe35d4ffa6fe7673179ad9d/shared-libs/cht-datasource/src/libs/doc.ts#L6) by contact. 

Or have there a contact's file, to return the contact summary and that would define the contact summary context

Reply:
i think i'll just add an analytics section.